docs/overview.rst

```diff
@@ -342,6 +342,59 @@ The contributions of each type of query to the total results displayed in the
 graph panel are displayed in different colors. These values can also be read
 directly by hovering over the relevant point on the time axis (the X axis).
 
+.. _overview-cluster-cloning:
+
+Cluster Cloning
+---------------
+
+Cluster cloning is a process of duplicating all the data from a
+specific snapshot into a different cluster. Creating the new cluster isn't
+part of the cloning process, you need to create a target cluster yourself. You
+can clone a cluster from the Backups page. 
+
+.. image:: _assets/img/cluster-backups.png
+   :alt: Cloud Console cluster backup snapshots
+
+Choose a snapshot and click the *Clone* button. A popup window will appear,
+where you can specify to which existing cluster the snapshot should be cloned.
+If you don't have a second cluster created, click the *Deploy a new cluster*
+button, and you will be brought to the cluster deployment screen. After you've
+selected a cluster, click the *Clone to selected cluster*. 
+
+.. image:: _assets/img/cluster-clone-popup.png
+   :alt: Cloud Console cluster clone popup
+
+.. NOTE::
+
+    Keep in mind that cloning a cluster will rewrite the all existing users
+    from the target cluster. The tables already existing on the target cluster
+    are not affected. Cloning also doesn't distinguish between cluster plans,
+    meaning you can clone from CR2 to CR1 or any other variation.
+
+.. _overview-cluster-cloning-fail:
+
+Failed cloning
+''''''''''''''
+
+There are circumstances under which cloning can fail. These are:
+
+* If you already have tables with the same names in the target cluster
+  as in the source snapshot. The entire clone operation will fail in that case.
+
+* There isn't enough storage left on the target cluster to accommodate the
```

Comment:
As far as I know, the cloning operation will not fail if there is not enough space on the target cluster - it will still succeed but just stop copying when the target cluster runs out of disk space